wled00/FX.cpp

```diff
@@ -1940,7 +1940,7 @@ uint16_t mode_palette() {
   using angleType = unsigned;
   constexpr mathType sInt16Scale             = 0x7FFF;
   constexpr mathType maxAngle                = 0x8000;
-  constexpr mathType staticRotationScale     = 256;
+  constexpr mathType staticRotationScale     = 255;
```

Comment:
thx for the review! really good point, did not think of that (I ran 1D tests mostly, as that is the issue addressed in this PR).

you are right, with mirroring this can be done also in 1D. So this was an unnecessary change.